src/site/layouts/vet_center.drupal.liquid

```diff
@@ -26,6 +26,53 @@
           <h2 id="locations-and-contact-information" class="vads-u-line-height--1 vads-u-margin-bottom--3 vads-u-font-size--lg
           small-screen:vads-u-font-size--xl" >Locations and contact information</h2>
 
+          <div class="region-list usa-grid usa-grid-full vads-u-display--flex vads-u-flex-direction--column
+          small-screen:vads-u-flex-direction--row facility vads-u-margin-bottom--2p5 vads-u-margin-bottom--4">
+              <div class="usa-width-two-thirds vads-u-display--block vads-u-width--full">
+
+                  <div>
+                      <div class="vads-c-facility-detail">
+                          <section class="vads-facility-detail">
+                                <h3 class="vads-u-font-size--lg vads-u-margin-top--0 vads-u-line-height--1 vads-u-margin-bottom--1">Main Location</h3>
+                                <div class="vads-u-margin-bottom--3">
+                                    <div>{{ fieldAddress.addressLine1 }}</div>
+                                    {{ fieldAddress.locality }},
+                                    {{ fieldAddress.administrativeArea }}
+                                    {{ fieldAddress.postalCode }}
+                                    <div><a
+                                         href="https://maps.google.com?saddr=Current+Location&daddr={{ fieldAddress.addressLine1 }}, {{ fieldAddress.locality }}, {{ fieldAddress.postalCode }}"
+                                         target="_blank" rel="noopener noreferrer">Directions
+                                        </a>
+                                    </div>
+                                </div>
+                                <h3 class="vads-u-font-size--lg vads-u-margin-top--0 vads-u-line-height--1 vads-u-margin-bottom--1">Direct line</h3>
+                                <div class="vads-u-margin-bottom--3">
+                                    <div class="main-phone vads-u-margin-bottom--1">
+                                        </strong><a href="tel:{{ fieldPhoneNumber }}">{{ fieldPhoneNumber }}</a>
+                                    </div>
+                                </div>
```

Comment:
Here's another refactor/reuse opportunity. This `<section class="vads-facility-detail">` block is copied/pasted form `src/site/layouts/health_care_local_facility.drupal.liquid`. Maybe we can clean this up later?